src/components/markdown-components/markdown-component.jsx

```diff
@@ -1,39 +1,108 @@
 import React , {Component} from 'react';
+import {Container} from 'react-bootstrap';
 
 import './markdown.css';
+class MarkDownComponent extends Component {   
+  constructor() {
+    super();
+    this.state= {
+      inputText: "",
+      markDownText: "",
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange=(e)=> {
+    this.setState({
+      inputText: e.target.value,  
+      },() =>{
+      this.makeBold(this.state.inputText)
+    });
+  };
+  makeBold=(inputText)=> {
+    // const {inputText} = this.state;
+    let boldStr= "";
+    let inputCopy= inputText;
+    let hashtag= "";
+    let mentionText= "";
+    let bgColor= "red";
+    let color= "blue";
+    let textColor="white";
+    let strikeText= "";
+    let italicText= "";
+    var regExp = /[a-zA-Z]/g; 
+    inputCopy.split("\n").forEach((str) => {
+      str.split(" ").forEach(elem =>{
+        const strLen = elem.length;
+        var firstIndexOfAsterisk = elem.indexOf("*",0);
+        var lastIndexAst = elem.lastIndexOf("*",strLen);
+        console.log(elem.substring(firstIndexOfAsterisk,firstIndexOfAsterisk+2));
+        //checking for ** tag 
+        if(
```

Comment:
it would be more visible 
```
        if('**' === elem.substring(firstIndexOfAsterisk,firstIndexOfAsterisk+2) &&
        "**" === elem.substring(lastIndexAst-1, lastIndexAst+1) && strLen>2) { 

```